test/project/PROJECT

```diff
@@ -1,3 +1,4 @@
 version: "1"
```

Comment:
What is this `version` used for? Have we used it somewhere?

Reply:
We used this to differentiate between the `pre-1.0` and `1.0` projects. We can probably use this field as `scaffoldingVersion` as well, I don't see any downside.

Reply:
Yes, I guess we can use `version` field for `scaffoldingVersion` purpose.

Reply:
Done.